awx/main/utils/common.py

```diff
@@ -212,6 +214,31 @@ def get_awx_version():
         return __version__
 
 
+def _get_validated_license_data():
+    return get_licenser().validate()
+
+
+def get_license(show_key=False):
```

Comment:
@ryanpetrello My previous implemetation (which imported get_license() from `awx.conf.license`) didn't work because it introduced a recursive import.  This commit fixes that by moved the method into `awx.main.utils.common`.